site/content/en/guides/consumer/get/_index.md

```diff
@@ -112,6 +145,7 @@ artifacts such as documentation.
 
 ### Package Contents Command
 
+<!-- @treePackage @verifyGuides-->
 ```sh
 tree cockroachdb/
```

Comment:
Is this command the only reason to install tree? It's simpler to use `ls cockroachdb/` and skip the install step in the verifyGuides workflow.

Reply:
I was trying to avoid changing the guides, but just make the guides executable as they currently are.

Reply:
This is actually exposing an implicit system requirement: user going through this guide needs to have tree installed.  Can we use 'kpt cfg tree' instead to remove this?

Reply:
Please follow up on this. My point is that you're ALSO asking the user to install `tree` command, but that's implicit and unnecessary IMO. As a general rule, impose as few dependencies on the user as possible.